File: api/foodpa/src/main/java/br/projeto/foodPa/model/Avaliacao.java
package br.projeto.foodPa.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

/**
 *
 * @author dougl
 */
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Avaliacao {
    private int id;
    private int nota;
    private int idProduto;
    private int idRestaurante;
    private int idUsuario;
   
}

File: foodpa v5/src/app/model/avaliacao.ts
export class Avaliacao {
    id: number;
    idRestaurante: number|null;
    idProduto: number|null;
    idUsuario: number;
    nota: number;

    constructor(){
        this.id =  0;
        this.idRestaurante = null;
        this.idProduto = null;
        this.idUsuario =  0;
        this.nota = 0;
    }
}
